Makefile

```diff
@@ -151,7 +151,7 @@ test: core
 all: dependencies core
 
 core:
-	@command -v $(MSBUILD) >/dev/null || (echo "OpenRA requires the '$(MSBUILD)' tool provided by Mono >= 5.4."; exit 1)
+	@command -v $(word 1,$(MSBUILD)) >/dev/null || (echo "OpenRA requires the '$(MSBUILD)' tool provided by Mono >= 5.4."; exit 1)
```

Comment:
What does this do?

Reply:
Replace `msbuild -verbosity:m -nologo` with `msbuild`.

Reply:
Ironically, despite both FreeBSD sh and NetBSD sh being Almquist shell descendants only FreeBSD is affected.